### DISCUSSION TITLE
Provide a global `css` tagged template literal function for better IDE DX

### DISCUSSION THREAD

**Discussion by alexilyaev** [johnlindquist/kit#1423] [Ideas]
Based on the example in `TIPS.md` > [Adjust the CSS of Choices](https://github.com/johnlindquist/kit/blob/main/TIPS.md#adjust-the-css-of-choices)

<details>
<summary>Expand code...</summary>

```ts
import "@johnlindquist/kit"

let choice = await arg({
  css: `
.light-purple {
  background-color: #c8a2c8;
}  
.medium-purple {
  background-color: #967bb6;
}
.dark-purple {
  background-color: #5d4777;
}

.focused {
  box-shadow: inset .5rem 0 0 0 #ffffffee;
}
  `,
  placeholder: "Choose a shade of purple",
  choices: [
    { name: "[L]ight Purple", value: "light-purple", className: "light-purple", focusedClassName: "focused" },
    { name: "[M]edium Purple", value: "medium-purple", className: "medium-purple", focusedClassName: "focused" },
    { name: "[D]ark Purple", value: "dark-purple", className: "dark-purple", focusedClassName: "focused" },
  ],
})

await div(md(`You chose ${choice}`))
```
</details>

The CSS code is just a plain string in VS Code:

<img width="665" alt="image" src="https://github.com/johnlindquist/kit/assets/480469/3ffa93e0-1f20-44a3-8767-f8bf621da2c0">

By adding a `css` tagged template literal function, we could leverage existing IDE tools like syntax highlighting, Emmet completion,  Prettier formatting and CSS intellisense via, e.g. [vscode-styled-components](https://marketplace.visualstudio.com/items?itemName=styled-components.vscode-styled-components) extension:

<img width="583" alt="image" src="https://github.com/johnlindquist/kit/assets/480469/4112edbf-3846-4766-ba48-fb55695b7ce7">
<img width="1184" alt="image" src="https://github.com/johnlindquist/kit/assets/480469/6810c99f-2ca1-4e1e-a01b-ee9177a1c5a8">

**Comment by johnlindquist** [comment]
@alexilyaev  That's a great suggestion, thanks.

In the meantime, it's trivial to write:

let css = await readFile(kenvPath("styles.css"), "utf8")

Then enjoy all the ide/tooling features in the styles.css file.
